vars/createSctRunner.groovy

```diff
@@ -21,6 +21,13 @@ def call(Map params, Integer test_duration, String region) {
     } else {
         availability_zone = params.availability_zone
     }
+
+    if ( params.backend.equals("azure") ) {
```

Comment:
Please, add this block of code after current line 33 to keep logical code blocks in one place.

Reply:
Fixed - moved it below the availability_zone blocks

Reply:
i don't see it moved... or i don't really understand where it was meant to...

Reply:
It was meant to kept the blocks related to `availability_zone` together and apart from the `region_zone` stuff